draft-thomson-quic-tls.md

```diff
@@ -724,6 +752,9 @@ server from being used in packet reflection attacks.
 
 A source address token is opaque and consumed only by the server.  Therefore it
 can be included in the TLS 1.3 pre-shared key identifier for 0-RTT handshakes.
+TODO: Could this be provided in TCP-based TLS 1.3 connections? If so that would
+mean that a TCP TLS 1.3 connection which was used to discover a server spoke
+QUIC could be used to enable a 0-RTT QUIC connection to that server.
```

Comment:
Ahh, you are looking to short circuit source address validation.  Source address isn't likely to be portable though, since the source address could change between the two protocols.  If you want to demonstrate control over a source IP, then that might work, but with CGN even that isn't guaranteed.  You could decide that a source address token from TCP increases confidence enough that you will risk transmitting toward an unwilling recipient.